stout/semaphore.h

```diff
@@ -2,13 +2,14 @@
 
 #ifdef __MACH__
 #include <mach/mach.h>
-#elif __WINDOWS__
+#elif _WIN32
 #else
```

Comment:
`#include <windows.h> // For Windows semaphore.`

And update line 54 to be `_WIN32` not `_WINDOWS_`.

Reply:
It is done.